lib/hammer_cli_csv/base.rb

```diff
@@ -756,7 +796,7 @@ def associate_locations(id, locations, name)
     def search_string(resource, name)
       operator = case resource
                  when "gpg-key", "sync-plan", "lifecycle-environment", "host-collections"
-                   @server_status['version'] && @server_status['version'].starts_with?('1.7') ? '=' : ':'
+                   @server_status['version'] && @server_status['version'].match(/^1\.7/) ? '=' : ':'
```

Comment:
prefer `\A` and `\z` over `^` and `$`, respectively. so this should be `match(/\A1\.7/)`.

[reference](http://stackoverflow.com/a/577675/336520)